mlserver/utils.py

```diff
@@ -13,11 +13,11 @@ async def get_model_uri(
     if not settings.parameters:
         raise InvalidModelURI(settings.name)
 
-    model_uri = settings.parameters.uri
-
-    if not model_uri:
+    relative_model_uri = settings.parameters.uri
```

Comment:
I think that the variable name `relative_model_uri` is slightly misleading here as it could also be absolute path?

perhaps also check directly whether it is absolute instead of relying on `normpath` (assuming that this is what happens currently which I am not 100% sure)

Reply:
We can call it `model_uri`? 

You're right in that it could also be an absolute path. In that case, `os.path.join()` should be the one handling it (it ignores the paths previous to an absolute path).